models/namespace.go

```diff
@@ -94,11 +91,8 @@ func CastProject(p osproject_v1.Project, cluster string) Namespace {
 	namespace.Cluster = cluster
 	namespace.CreationTimestamp = p.CreationTimestamp.Time
 	namespace.Labels = p.Labels
-	namespace.Annotations = make(map[string]string)
-	// Parse only annotations used by Kiali
-	if da, ok := p.Annotations[dashboards.DashboardTemplateAnnotation]; ok {
-		namespace.Annotations[dashboards.DashboardTemplateAnnotation] = da
-	}
+	namespace.Annotations = p.Annotations
```

Comment:
I have the same change for my work of indicating which namespace is primary and which are remote. I think this is necessary for my work of hiding some information in the control plane card. I'm detecting remotes with the annotation `topology.io/controlPlaneClusters`.